### ISSUE TITLE
Wrong Exif interpretation on Android using Image

### ISSUE THREAD

**Issue by adamczyk777** [facebook/react-native#19647]
- [x] Review the documentation: https://facebook.github.io/react-native
  - [x] Search for existing issues: https://github.com/facebook/react-native/issues
  - [x] Use the latest React Native version: https://github.com/facebook/react-native/releases
  - [x] Run `react-native info` in your terminal and paste its contents under "Environment"
  - [x] Let us know how to reproduce the issue. Include a code sample, share a project, or
    share an app that reproduces the issue using https://snack.expo.io/

## Environment
Environment:
  OS: macOS High Sierra 10.13.4
  Node: 9.5.0
  Yarn: 1.6.0
  npm: 5.6.0
  Watchman: 4.9.0
  Xcode: Xcode 9.4 Build version 9F1027a
  Android Studio: 3.1 AI-173.4720617

Packages: (wanted => installed)
  react: 16.3.1 => 16.3.1
  react-native: ~0.55.2 => 0.55.4

## Description
There is a difference between how iOS and Android Image component handles exif. I was searching through issues from past, and the issue of wrong orientation was to be resolved, but it is still happening in my app.  Photos are being served from devices memory on both iOS and Android. I already checked out that on Android Image component is handled by fresco facebooks android library, and the fixes from before were applied to newest react native version.

## Steps to Reproduce
Use Image component, To be specific I use Animated.Image, load local device image uri as source to it. You can use same images as I do [testing samples](https://github.com/ianare/exif-samples/blob/master/jpg/orientation/landscape_1.jpg).

To Test it by yourself make a new app with create react native app and paste those in the View in App.js the behavior is exactly the same with photos fetched from the internet, and with the lastest version of React Native (as you can see to develop our app we currently use 0.53 not yet updated)
If you run on both android and ios the behavior is not consistent.

```
<Image style={{width: 50, height: 50}} source={{uri: 'https://raw.githubusercontent.com/ianare/exif-samples/master/jpg/orientation/landscape_1.jpg'}}/>
<Image style={{width: 50, height: 50}} source={{uri: 'https://raw.githubusercontent.com/ianare/exif-samples/master/jpg/orientation/landscape_2.jpg'}}/>
<Image style={{width: 50, height: 50}} source={{uri: 'https://raw.githubusercontent.com/ianare/exif-samples/master/jpg/orientation/landscape_3.jpg'}}/>
<Image style={{width: 50, height: 50}} source={{uri: 'https://raw.githubusercontent.com/ianare/exif-samples/master/jpg/orientation/landscape_4.jpg'}}/>
<Image style={{width: 50, height: 50}} source={{uri: 'https://raw.githubusercontent.com/ianare/exif-samples/master/jpg/orientation/landscape_5.jpg'}}/>
<Image style={{width: 50, height: 50}} source={{uri: 'https://raw.githubusercontent.com/ianare/exif-samples/master/jpg/orientation/landscape_6.jpg'}}/>
<Image style={{width: 50, height: 50}} source={{uri: 'https://raw.githubusercontent.com/ianare/exif-samples/master/jpg/orientation/landscape_7.jpg'}}/>
<Image style={{width: 50, height: 50}} source={{uri: 'https://raw.githubusercontent.com/ianare/exif-samples/master/jpg/orientation/landscape_8.jpg'}}/>
```

## Expected Behavior
Expected behavior is following:
Images display in position where numbers area readable and not rotated or mirrored, on iOS I get that behavior
Screen from iOS
![image](https://user-images.githubusercontent.com/16062886/41218446-5fa6b852-6d5b-11e8-9792-54338f2c6ff3.png)

## Actual Behavior
Actual behavior is following:
Exif numbers 7, 4, 5, 2 appear to be wrong interpreted, while 1,3,8,6 are all right. Android screen:
![image](https://user-images.githubusercontent.com/16062886/41218519-b25ad998-6d5b-11e8-8cbf-36a33ed6bf5d.png)

**Comment by adamczyk777** [comment]
Bot Assigned a wrong label it is reproduced in the latest version also, but main main experience comes from not the recent version

**Comment by adamczyk777** [comment]
Updating Fresco dependency in ReactAndroid to version 1.9.0 helped

**Comment by adamczyk777** [comment]
As far as I can see problem is going to be fixed by RN 0.56 release. 0.56-rc added fresco 1.9.0. If it works out Issue can be closed

**Comment by hramos** [comment]
Looks like this is fixed in master and in `0.56.0-rc`. Closing as fixed.

**Comment by phillmill** [comment]
@adamczyk777 thank you. I knew I wasn't the only one who has this issue! I find it odd there haven't been many reports as this is a bug right out of the box.

Note I also opened a new bug here about this here a couple months ago: https://github.com/facebook/react-native/issues/20125
